src/content/getting-started/prepare-your-provider-account/azure/_index.md

```diff
@@ -70,15 +70,15 @@ Alternatively, you can create your role assignment with restricted access to spe
 
 * Now you can run this command to create the Deployment Environment:
 
-    ```sh
-    az deployment create --name "giantswarm-access" \
-                         --location <AzureRegion> \
-                         --template-file delegatedResourceManagement.json \
-                         --parameters delegatedResourceManagement.parameters.json \
-                         --verbose
-    ```
+```sh
+$ az deployment create --name "giantswarm-access" \
+                      --location <AzureRegion> \
+                      --template-file delegatedResourceManagement.json \
+                      --parameters delegatedResourceManagement.parameters.json \
+                      --verbose
+```
```

Comment:
This was meant to be indented as part of the bullet point list

````suggestion
    ```sh
    $ az deployment create --name "giantswarm-access" \
                          --location <AzureRegion> \
                          --template-file delegatedResourceManagement.json \
                          --parameters delegatedResourceManagement.parameters.json \
                          --verbose
    ```
````